community/2023/community-04-24.md

```diff
@@ -0,0 +1,19 @@
+# March 28th Community stream
+
+**See the [instructions](../README.md) for details on how to attend**
+
+6PM PST on the last Tuesday of the month.
```

Comment:
I realize this means 6pm on this date in  the US Pacific time zone, which happens to be PDT at this point not PST, but I never know whether people take "PST" literally as "Pacific Standard Time" or just "Whatever Pacific Time it is on this date".  Time zones can be such a pain...